### PR TITLE
plotter: add examples for labels

### PR DESCRIPTION
**Pull request by sbinet**
Fixes gonum/plot#568.

Please take a look.

### CODE REVIEW

File: plotter/labels.go [hunk 1]
```diff
@@ -122,3 +122,7 @@ type XYLabels struct {
 func (l XYLabels) Label(i int) string {
 	return l.Labels[i]
 }
+
+var (
+	_ XYLabeller = (*XYLabels)(nil)
+)
```
Comment (kortschak) [inline plotter/labels.go:128]: `var _ XYLabeller = (*XYLabels)(nil)`

File: plotter/example_label_test.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	defer f.Close()
+
+	cnv := vgimg.PngCanvas{
+		Canvas: vgimg.New(10*vg.Centimeter, 10*vg.Centimeter),
+	}
+
+	dc := draw.New(cnv)
+	p.Draw(dc)
+
+	// put an 'X' in the middle of the data-canvas
```
Comment (kortschak) [inline plotter/example_label_test.go:93]: `// Put an 'X' in the middle of the data-canvas.`

File: plotter/example_label_test.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+	p.Add(labels)
+
+	err = p.Save(10*vg.Centimeter, 10*vg.Centimeter, "testdata/labels.png")
+	if err != nil {
+		log.Fatalf("could save plot: %+v", err)
+	}
+}
+
+func ExampleLabels_inCanvasCoordinates() {
```
Comment (kortschak) [inline plotter/example_label_test.go:52]: Can this have an explanatory preamble?

**Comment by sbinet** [comment]
PTAL

**Comment by sbinet** [comment]
ping?

**Pull request merged**